### PR TITLE
[BUD-33] Contact Details Screen

### PR DESCRIPTION
**Pull request by michaelmoney**
# [BUD-33](https://digitalrig.atlassian.net/browse/BUD-3) Contact Details Screen

## Description

Display newbie Contact Details

image

### CODE REVIEW

File: src/components/ContactDetails/ContactDetails.tsx [hunk 1]
```diff
@@ -1,13 +1,126 @@
...
+  },
+  ellipse: {
+    position: 'fixed',
+    bottom: '-285px',
+    left: '-56px',
+  },
+  wrapper: {
+    display: 'flex',
+    flexDirection: 'row',
+    marginTop: '2rem',
```
Comment (lucianojss) [inline src/components/ContactDetails/ContactDetails.tsx]: You can use here `theme.spacing(x)` function :)

File: src/components/ContactDetails/ContactDetails.tsx [hunk 2]
```diff
@@ -1,13 +1,126 @@
...
+import Avatar from '../Avatar';
+import { ReactComponent as EllipseShape } from '../../svg/ellipse-shape-gray.svg';
+import { ROUTES } from '../../shared/routes';
+import { ContactDetailsParams, ContactDetailsProps, NewbieData } from './types';
 
-const ContactDetails: React.FC = () => {
-  const { id: newbieId } = useParams();
+const useStyles = makeStyles(theme => ({
+  bold: {
+    fontWeight: 700,
```
Comment (lucianojss) [inline src/components/ContactDetails/ContactDetails.tsx]: We can follow the design tokens defined in the teme to pick different font weights.
For example, taking a look at https://material-ui.com/system/typography/, we can verify that we have available following fontWeights:

- fontWeightLight: 300
- fontWeightRegular: 400
- fontWeightMedium: 500
- fontWeightBold: 700

File: src/components/ContactDetails/ContactDetails.tsx (hunk 1 above)
Comment (michaelmoney) [reply src/components/ContactDetails/ContactDetails.tsx]: Done here: https://github.com/BuildIt-Poland/buddy-app/pull/21/commits/d7412a4124e56f72710eb43e6ff3f3286905d1d5 and here: https://github.com/BuildIt-Poland/buddy-app/pull/21/commits/d84311a66dd86508fad577396a9983ac43d8dd92

File: src/svg/ellipse-shape-gray.svg [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<svg width="650" height="454" viewBox="0 0 650 454" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (lucianojss) [inline src/svg/ellipse-shape-gray.svg]: Can you remove this  `.svg` it's not going to be needed I will had the ellipse component to this page afterwards 👍

File: src/components/ContactDetails/__tests__/ContactDetails.test.tsx [hunk 4]
```diff
@@ -1,18 +1,65 @@
...
+          <Route path={'/buddy/newbies/:newbieId/details'}>
+            <ContactDetails history={mockHistory} />
+          </Route>
+        </MemoryRouter>
+      </MockedProvider>
     );
+    expect(component).toMatchSnapshot();
 
-    expect(component.toJSON()).toMatchSnapshot();
+    await act(async () => {
```
Comment (lucianojss) [inline src/components/ContactDetails/__tests__/ContactDetails.test.tsx]: art piece of engineering 🚀 😄

**Pull request merged**